src/styles/components/_card.scss

```
@@ -0,0 +1,30 @@
+@use '../abstracts/functions' as *;
+@use '../abstracts/variables' as variables;
+@use '../abstracts/mixins' as *;
+
+.card {
+    @include flex(
+        $justify: center,
+        $align: center,
+        $direction: column,
+        $gap: px-to-rem(29)
+    );
+    padding: px-to-rem(32);
+    border: 1px solid color-opacity(map-get(variables.$colors, grey-black), 0.1);
+    border-radius: px-to-rem(32);
```

Can we add a slight transition for the hover state. ?? 
```suggestion
    transition: box-shadow 0.3s ease-in-out;
```